jina/helper.py

```diff
@@ -662,7 +660,8 @@ def get_full_version() -> Tuple[Dict, Dict]:
         env_info = {k: os.getenv(k, '(unset)') for k in __jina_env__}
         return info, env_info
     except Exception as e:
-        default_logger.error(e)
+        default_logger.error(str(e))
+        raise
```

Comment:
if it was not there, better avoid raising, I guess the process will continue working even when this error occured so do not want to interrupt it

Reply:
Fair enough. I'll change this function to return `Optional[...]`.